frontend/src/router.tsx

```diff
@@ -0,0 +1,92 @@
+import { createBrowserRouter } from "react-router-dom";
+
+import {DE_WIEKEN, getProjectConfiguration, HESSENPOORT, ProjectName} from "./components/company-survey-v2/project"
+import {SurveyFromProject} from "./components/company-survey-v2/survey"
+import {ThankYou} from './components/thank-you'
+import {LoginWidget} from "./user/login";
+import {BedrijvenFormV1} from "./components/bedrijven-form-v1";
+import {Admin} from "./admin/admin";
+import {fetchSurveyById, SurveyById, SurveyByIdLoaderData} from "./components/company-survey-v2/survey-by-id"
+import {Intro} from "./components/intro"
+import {ExcelImport} from "./excel-import/excel-import"
+import {NewSurveyByProjectName} from "./components/company-survey-v2/new-survey-by-project-name"
+import {assertDefined} from "./services/util"
+import Simulation from "./simulation";
+import {App} from "./App";
+
+export const router = createBrowserRouter([
+    {
+        path: "/",
+        element: <App />,
+        children: [
+            {path: "", element: <Intro />},
+            {path: "/simulation", element: <Simulation />},
+        ],
+    },
```

Comment:
I don't understand the point of this component.

Why are there child routes? AFAIK the reason for child routes is to share a layout. `<App />` does not have a layout.

Reply:
Nvm I see you add a layout in the next PR

Reply:
;)